setup/general/kuksa.val/generate-config.sh

```diff
@@ -93,6 +93,6 @@ echo "### Modified file: $CONFIG_DIR/vss-mod.json"
 echo
 
 echo "Execute the following command to install modified vss.json"
-echo "  cp vss-mod.json ../../../docker-compose/config/kuksa.val/vss.json"
-echo "  cp -r certs ../../../docker-compose/config/kuksa.val/"
+echo "  cp config/vss-mod.json ../../../docker-compose/config/kuksa.val/vss.json"
```

Comment:
Use the $CONFIG_DIR variable instead of specifying it directly.
`echo "  cp ${CONFIG_DIR}/vss-mod.json ../../../docker-compose/config/kuksa.val/vss.json"`
The same goes for line 97

Reply:
Updated.